mne/beamformer/tests/test_lcmv.py

```diff
@@ -288,6 +288,14 @@ def test_lcmv():
     assert 0.08 < tmax < 0.11, tmax
     assert 0.4 < np.max(max_stc) < 2., np.max(max_stc)
 
+    # Test if spatial filter contains src
+    assert ('src' in filters.keys())
+
+    # Test if stc contains src
+    with warnings.catch_warnings(record=True) as wrn:
+        stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+    assert not any('src should not be None' in str(ww.message) for ww in wrn)
+
```

Comment:
@agramfort would this work? If yes, I will implement the same in ``test_dics``

Reply:
perfect or even better can you do this around an already existing call to apply_lcmv to test without even increasing testing time?

Reply:
Ah! Of course, makes perfect sense!